oml/lightning/pipelines/logging.py

```diff
@@ -31,6 +34,66 @@ def prepare_tags(cfg: TCfg) -> List[str]:
     return tags
 
 
+class ClearMLLogger(IPipelineLogger):
```

Comment:
not sure about inheritance here

I think `ClearMLLogger` should be a child of `LightningLogger`

and `ClearMLPipelineLogger` should be a child of `ClearMLLogger` and `IPipelineLogger`

Am I wrong?

Reply:
You're right. I'll fix it.

Reply:
Done.